src/deluge/gui/views/view.cpp

```diff
@@ -751,8 +752,16 @@ void View::clearMelodicInstrumentMonoExpressionIfPossible() {
 	}
 }
 
+/* adapt program change messages to cc*/
+void View::pcReceivedForMIDILearn(MIDIDevice* fromDevice, int32_t channel, int32_t pc) {
```

Comment:
I will remove this, thanks for pointing it out. 

and had this been the actual code path that i needed to implement, i might have argued that using note on for CC doesn't really work either, its admittedly a bit of a hack. 

An additional reason that I will remove this, is because old midi controllers have noise/mechanism issues which cause intermittent stray CC's to be sent. Sometimes constantly. Program changes less so.

Reply:
we went over this and re-iterated why we are removing this and removed it.